examples/mixture_models/dirichlet_mixture_of_multinomials.ipynb

```diff
@@ -0,0 +1,748 @@
+{
```

Comment:
<p>Same thing here: </p><ul><li>What's the model doing? </li><li>What do the parameters mean? </li><li>How are you choosing the priors?</li><li>Why are you parametrizing the Dirichlet with <code>a * alpha</code> instead of the classical <code>alpha</code> ? </li><li>Are you expecting this model to run well?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>